source/resource/visu_config.xsd

```diff
@@ -413,10 +413,11 @@
         <xsd:enumeration value="cgi-bin" />
         <xsd:enumeration value="oh" />
         <xsd:enumeration value="oh2" />
+        <xsd:enumeration value="openhab2" />
         <!-- New values, should be used now -->
         <xsd:enumeration value="default" />
+        <xsd:enumeration value="knxd" />
```

Comment:
As default == knxd we should also move default to the deprecated section, right?